src/main/kotlin/uk/gov/gdx/datashare/service/EventDataService.kt

```diff
@@ -85,18 +89,20 @@ class EventDataService(
     } ?: consumerSubscriptionRepository.findAllByClientId(clientId)
 
     if (consumerSubscriptions.isEmpty()) {
```

Comment:
Unrelated to this change, but I'm somewhat sad this is required, you'd have hoped any of the later queries/maps would go "empty in, empty out"